packages/theme/css/foundations/shadow.css

```diff
@@ -15,9 +15,9 @@
 }
 
 .salt-theme {
-  --salt-shadow-1: 0 1px 3px 0 var(--salt-shadow-1-color);
-  --salt-shadow-2: 0 2px 4px 0 var(--salt-shadow-2-color);
-  --salt-shadow-3: 0 4px 8px 0 var(--salt-shadow-3-color);
-  --salt-shadow-4: 0 6px 10px 0 var(--salt-shadow-4-color);
-  --salt-shadow-5: 0 12px 40px 0 var(--salt-shadow-5-color);
+  --salt-shadow-100: 0 1px 3px 0 var(--salt-shadow-1-color);
+  --salt-shadow-200: 0 2px 4px 0 var(--salt-shadow-2-color);
+  --salt-shadow-300: 0 4px 8px 0 var(--salt-shadow-3-color);
+  --salt-shadow-400: 0 6px 10px 0 var(--salt-shadow-4-color);
+  --salt-shadow-500: 0 12px 40px 0 var(--salt-shadow-5-color);
```

Comment:
need to change the colors to match (design dont have these tokens, theyre to simplify the code)

Reply:
yup these were addressed in later commits - will resolve this comment for now